packages/web-components/src/components/accordion/accordion.scss

```
@@ -93,9 +132,41 @@ $css--plex: true !default;
 :host(#{$prefix}-accordion-item-skeleton) {
   .#{$prefix}--accordion__heading {
     cursor: default;
+    pointer-events: none;
 
     &:hover::before {
       background: transparent;
     }
   }
 }
+
+:host(#{$prefix}-accordion-item[alignment='end' i]),
+:host(#{$prefix}-accordion-item-skeleton[alignment='end' i]),
+:host(#{$prefix}-accordion-skeleton)
+  .#{$prefix}--accordion__item.#{$prefix}--accordion--END {
+  .#{$prefix}--accordion__arrow {
+    margin-inline-start: $spacing-05;
+  }
+}
+
+:host(#{$prefix}-accordion-item-skeleton) .#{$prefix}--accordion__title,
+:host(#{$prefix}-accordion-skeleton)
+  .#{$prefix}--accordion__heading
+  .#{$prefix}--accordion__title {
+  padding-block-start: $spacing-03;
+}
+
+.controlled-accordion-btnset {
+  @extend .cds--btn-set !optional;
+
+  padding-block-end: $spacing-03;
+
+  .controlled-accordion-btn:not(:first-child)::part(button) {
+    box-shadow: -0.0625rem 0 0 0
+      var(--cds-button-separator, --cds-border-subtle);
```

I think you can get this one from the button component tokens

```diff
+ @use '@carbon/styles/scss/components/button/tokens' as *;
```

```suggestion
      $button-separator, $border-subtle;
```

---

Thank you!!